src/components/templates/ConversationSpace/ConversationSpace.scss

```diff
@@ -315,3 +320,22 @@
     }
   }
 }
+
+.conversation-space-background {
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  background-color: opaque-black(0.8);
+}
+
+.TableHeader {
+  &__leave-table {
+    z-index: z(jazzbar-header);
+  }
+  &__lock-button {
+    z-index: z(jazzbar-header);
+  }
+  &__topic-info {
+    z-index: z(jazzbar-header);
+  }
+}
```

Comment:
why do we need z-indexes now?

Reply:
ref https://github.com/sparkletown/sparkle/pull/2265#discussion_r708476609